Mathlib/MeasureTheory/PiSystem.lean

```diff
@@ -106,10 +106,7 @@ theorem IsPiSystem.comap {α β} {S : Set (Set β)} (h_pi : IsPiSystem S) (f : 
   rintro _ ⟨s, hs_mem, rfl⟩ _ ⟨t, ht_mem, rfl⟩ hst
   rw [← Set.preimage_inter] at hst ⊢
   refine' ⟨s ∩ t, h_pi s hs_mem t ht_mem _, rfl⟩
-  by_contra h
-  rw [Set.not_nonempty_iff_eq_empty] at h
-  rw [h] at hst
-  simp at hst
+  exact nonempty_of_nonempty_preimage hst
```

Comment:
Looks like you can even collapse this into the previous line

Reply:
Done in https://github.com/leanprover-community/mathlib4/pull/11052/commits/9b8d5adfce2abede54d70e1444a329c0a337fac3.